custom/icds_reports/migrations/sql_templates/database_views/child_health_monthly.sql

```diff
@@ -10,7 +10,9 @@ CREATE VIEW child_health_monthly_view AS
       awc_location_months.block_id,
       awc_location_months.block_name,
       awc_location_months.district_id,
+      awc_location_months.district_name,
```

Comment:
you will also need to add the migration file for ths.

Reply:
added